### PR TITLE
[U] Enhancements to English Translations and Standardization to British English

### PR DESCRIPTION
**Pull request by cvyl**
Firstly, I want to express my deep appreciation for the incredible work you have done with this website. Reading the stories on this site has profoundly moved me, and I have shed tears multiple times. The memorials serve as a powerful reminder of the struggles of the transgender community, and your dedication to preserving these memories is commendable.

I have made several updates to the English translations on your site. While I am not a native Chinese speaker and possess only a very basic understanding of Mandarin Chinese, I have done my utmost to enhance the English content. Here are the details of my contributions:

- Improved translation and corrected non-existent words.
- Fixed various spelling mistakes.
- Standardised capitalisation and grammar.
- Updated American English wording to British English for consistency.

Rationale for British English:
I believe British English is a more appropriate standard for the site, as it is widely understood globally and provides a sense of formality and respect that aligns well with the solemn and respectful nature of a memorial website. British English is also commonly used in international publications, which may help in reaching a broader audience.

Intentional Retention of Original Wording:
On some pages, I have intentionally not fixed certain grammatical errors or mistranslations. I felt that changing too many words of the deceased could be seen as disrespectful, so I kept these alterations to a very minor amount to preserve the original sentiment as much as possible.

If you need any further feedback or revisions, please do not hesitate to reach out. I am eager to contribute to this meaningful project and am open to any suggestions or additional changes you might have.

### CODE REVIEW

**Comment by Twinsherry** [comment]
Amika在此先谢过您的工作。

Thanks for your works.

稍晚时我们的其他编辑者会在本Pull Request下方留言。

Other editors would comment in this Pull Request later on.

File: people/CuspariaKLSY/page.en.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ If she believes sacrificing herself would benefit others, she won't have a secon
 
 She cares about her friends a lot.
 
-She's a Eastern Orthodox Christian.
+She's an Eastern Orthodox Christian.
```
Comment (Aquaticat) [inline people/CuspariaKLSY/page.en.md:32]: Right...

File: people/Dethelly/page.en.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ Moreover, she hadn't a friend to confide her thought, she could only hide these
 That made her look a bit intimidated.
 
 She always prepared a lot ahead of meeting friends in real life.
-She was always worried about leaving a bad impression, so she asked them tons of questions in advance (such as which kinds of gift they would like), and warned them she might exhibit regressive behaviors.
+She was always worried about leaving a bad impression, so she asked them tons of questions in advance (such as which kinds of gift they would like), and warned them she might exhibit regressive behaviours.
```
Comment (Aquaticat) [inline people/Dethelly/page.en.md:18]: Prefer en_US or en_CA over en_GB.

File: people/Dethelly/page.en.md [hunk 3]
```diff
@@ -56,13 +56,13 @@ That night, Sugar actually deleted her contact.
...
 
-Later, Sugar began her winter vacation, and got her diagnosis certificate of gender dysphoria in Shenzhen Kangning Hospital —— This was only known by A after her death.
+Later, Sugar began her winter holiday, and got her diagnosis certificate of gender dysphoria in Shenzhen Kangning Hospital —— This was only known by A after her death.
 
 ## Thick fog over Jialing River (Her departure)
 
 On January 22nd, 2024, she went back to her hometown, Chongqing, and regained contact with A.
 
-(According to her mother who worked away from Chongqing, when she just returned home, she seemed happy in Wechat.)
+(According to her mother who worked away from Chongqing, when she just returned home, she seemed happy in WeChat.)
```
Comment (Aquaticat) [inline people/Dethelly/page.en.md:65]: Thanks for catching that.

File: people/Dethelly/page.en.md [hunk 4]
```diff
@@ -56,13 +56,13 @@ That night, Sugar actually deleted her contact.
 The second day, a Twitter user who claimed to be Sugar's high school classmate brought her some random thoughts on behalf of Sugar.
 Then A requested them to look after her.
 
-Later, Sugar began her winter vacation, and got her diagnosis certificate of gender dysphoria in Shenzhen Kangning Hospital —— This was only known by A after her death.
+Later, Sugar began her winter holiday, and got her diagnosis certificate of gender dysphoria in Shenzhen Kangning Hospital —— This was only known by A after her death.
```
Comment (Aquaticat) [inline people/Dethelly/page.en.md:59]: May I ask why?

File: people/Dethelly/page.en.md [hunk 5]
```diff
@@ -134,7 +134,7 @@ In a daze, a phone call from Chongqing came, and she was familiar with the numbe
 > Girl, I am glad you're okay, wish you a happy life.
 
 Sugar had a very cute face and figure.
-If we change a character of her name to another homophonic one, it would became a girl's name that perfectly matches her gentle and soft temperament.
+If we change a character of her name to another homophonic one, it would become a girl's name that perfectly matches her gentle and soft temperament.
```
Comment (Aquaticat) [inline people/Dethelly/page.en.md:137]: Thanks for catching the error in tense.

File: people/Futajuhuacha/page.en.md [hunk 6]
```diff
@@ -11,16 +11,16 @@ Huacha was a lovely and determined trans girl who loved this world.
 
 She was an OIer and once participated in the National Olympiad in Informatics(NOI).
 
-She adored long hair, so she opted for hair extension to fulfill her love for it.
+She adored long hair, so she opted for hair extension to fulfil her love for it.
```
Comment (Aquaticat) [inline people/Futajuhuacha/page.en.md:14]: Prefer en_US or en_CA over en_GB.

File: people/Dethelly/page.en.md [hunk 7]
```diff
@@ -170,7 +170,7 @@ It was a pity that she departed abruptly, without waiting for a turn for the bet
 > 
 > Hope that if you have the ability to call the police, you can dial decisively. You are a lifesaver, not a murderer. Don't be afraid.
 > 
-> Hope you cherishes life, hope still exists; you will only miss the opportunity if you leave too early.
+> Hope you cherish life, hope still exists; you will only miss the opportunity if you leave too early.
```
Comment (Aquaticat) [inline people/Dethelly/page.en.md:173]: Thanks for catching the error.

File: people/GLaDOSister/page.en.md [hunk 8]
```diff
@@ -20,17 +20,17 @@ She had contributed to the community of the *Half-Life* series.
 She designed some "GMod" plugins and maps; and she was the director of a fanmade work called *Combine Prelude* for *Half-Life 2*.
 The fact that worth mentioning the most is, she started doing constructed language research,
 constructed [Combinese](https://yuyan.fandom.com/zh/wiki/联合军语), [Yupoian](https://yuyan.fandom.com/zh/wiki/尤波伊语) and [Crigàt](https://yuyan.fandom.com/zh/wiki/克里加语) for the project mentioned above, *Combine Prelude*.
-Because of her work on this project, she was famous both in the *Half-Life* community and in the contructed language community.
+Because of her work on this project, she was famous both in the *Half-Life* community and in the constructed language community.
 
-Shizuha's favorite anime is *YuruYuri*.
+Shizuha's favourite anime is *YuruYuri*.
```
Comment (Aquaticat) [inline people/GLaDOSister/page.en.md:25]: Prefer en_US or en_CA over en_GB.

File: people/Mio/page.en.md [hunk 9]
```diff
@@ -62,19 +62,19 @@ Translation:
...
 > 
 > Me: I think we need to meet. How about May 1st? (Translator's note: a holiday)
 > 
-> Me: (photo, a Wechat post)
+> Me: (photo, a WeChat post)
 > 
 > > (Translation of the post):
 > >
-> > To be able to maintain a state between male and female is a miracle――height, body shape, behavior, skin, and voice.
+> > To be able to maintain a state between male and female is a miracle――height, body shape, behaviour, skin, and voice.
```
Comment (Aquaticat) [inline people/Mio/page.en.md:77]: Prefer en_US or en_CA over en_GB.

File: people/MizuharaNagisa/page.en.md [hunk 10]
```diff
@@ -12,11 +12,11 @@ She was friends with [Ying](https://www.one-among.us/profile/Uekawakuyuurei/).
 During her life, she was an active long-term contributor for Chinese Wikipedia and other MediaWiki sites.
 Since 2019, she started playing *Kancolle*.
 Her favourite character was Destroyer Hibiki.
-Her hobbies were tinkering with computer hardware like Respberry Pi and drawing.
+Her hobbies were tinkering with computer hardware like Raspberry Pi and drawing.
 She was also proficient in Japanese.
 
 Because of depression and other problems, she had to stop going to school since 2019;
-because of inacceptance of her family and other problems she faced, her depression and anxiety got worse and started to suffer from insomnia.
+because of lack of understanding from her family and other problems she faced, her depression and anxiety got worse and started to suffer from insomnia.
```
Comment (Aquaticat) [inline people/MizuharaNagisa/page.en.md:19]: Should've been unacceptance.

File: people/Mizuki_Yuuki/page.en.md [hunk 11]
```diff
@@ -22,8 +22,8 @@ My heart was filled with endless mourning and sorrow.
 In early December, when Mizuki got their first Kigurumi headpiece, they were overjoyed and felt liberated.
 Three days later, I invited them for an outdoor photoshoot in Century Park – my first outdoor photoshoot.
 
-After this photoshoot, Mizuki organized a Kigurumi party and traveled twice: one to Kuala Lumpur before Christmas, and another to Hong Kong after New Year's Day.
-They were not fond of traveling so that was a bit surprising.
+After this photoshoot, Mizuki organized a Kigurumi party and travelled twice: one to Kuala Lumpur before Christmas, and another to Hong Kong after New Year's Day.
```
Comment (Aquaticat) [inline people/Mizuki_Yuuki/page.en.md:25]: Prefer en_US or en_CA over en_GB.

File: people/Mizuki_Yuuki/page.en.md [hunk 12]
```diff
@@ -22,8 +22,8 @@ My heart was filled with endless mourning and sorrow.
...
+They were not fond of travelling, so that was a bit surprising.
```
Comment (Aquaticat) [inline people/Mizuki_Yuuki/page.en.md:26]: Prefer en_US or en_CA over en_GB.

File: people/Xu_Yushu/page.en.md [hunk 13]
```diff
@@ -20,14 +20,14 @@ Maybe it's more accurate to describe her as *literary girl*.
...
-Xu Yushu had always been a lonely child but she made friends with books.
+Xu Yushu had always been a lonely child, but she made friends with books.
 Reading made her mature faster than her peers.
 She loved being immersed in the world of books.
 Her bookshelf was filled up already at that time.
 Not only that, she also went to the bookstore next to her father's workspace and left a deep impression on the bookstore's manager.
 
 During her high school years, Nan'an Wanda Plaza's Sisyphe Books was one of the places she went to the most.
-There, she could find tranquility in the pleasant smell of coffee and books.
+There, she could find tranquillity in the pleasant smell of coffee and books.
```
Comment (Aquaticat) [inline people/Xu_Yushu/page.en.md:30]: Prefer en_US or en_CA over en_GB.

File: people/yumao/page.en.md [hunk 14]
```diff
@@ -33,7 +33,7 @@ She made wishes to meteors occasionally. She said: "It's not because I believe t
 According to Neko, Yumao loved spicy hot pot, and always enjoyed it with rice.
 
 Yumao liked to watch stars while waiting to fall asleep at night.
-After she returned to China, due to bad weather and few spare time, she couldn't watch stars as often as she did.
+After she returned to China, due to bad weather and fewer spare time, she couldn't watch stars as often as she did.
```
Comment (Aquaticat) [inline people/yumao/page.en.md:36]: "bad weather" is not a comparison, therefore "? spare time" cannot be a comparison as well.

File: people/yingying/page.en.md [hunk 15]
```diff
@@ -1,13 +1,16 @@
...
+## Description
+
 Yingying is a trans woman who had undergone sex reassignment surgery. She was a lively and cheerful girl who was interested in electronics and chemistry. She looks very handsome when she rides a bicycle. She always brought happiness to those around her. She had a toothpaste box with the Intel Logo at home. She said that every time people enter her bathroom, that toothpaste box made them laugh. [^1]
 
 She had published a lot of articles and videos about electronics in both her personal Bilibili account and the Bilibili account “Geek Lab” she created. She had also posted some videos about motorcycle riding. She also opened a store on Taobao where she sold and rented out electronics. She hoped to go to Guangzhou to develop her career.
 
 It's a pity that all this stopped abruptly when she lost her life forever in a car accident.
 
-May she still have the same cheerfulness and passion to do what she want to do in her next world! [^1]
+May she still have the same cheerfulness and passion to do what she wanted to do in her next world! [^1]
```
Comment (Aquaticat) [inline people/yingying/page.en.md:13]: Should this really be past tense?
I felt like the meaning is about what she wants to do in the future.

File: people/qiqi233345/page.en.md [hunk 16]
```diff
@@ -1,22 +1,22 @@
 ---
 name: Qi
 ---
-Qi didn’t often talk on the Internet, but she was very welcoming in real life. Her humor always brought people around her happiness.
+Qi didn’t often talk on the Internet, but she was very welcoming in real life. Her humour always brought people around her happiness.
```
Comment (Aquaticat) [inline people/qiqi233345/page.en.md:4]: Prefer en_US or en_CA over en_GB.

File: people/qiqi233345/page.en.md [hunk 17]
```diff
@@ -1,22 +1,22 @@
...
+On March 14, 2023, she attempted suicide by jumping off a building due to depression, and her legs were disabled. This made her life very inconvenient, and she became more depressed.
 
 On July 1, 2023, she jumped off a building again after taking a large amount of antidepressant drugs.
 This time, her injuries were beyond saving.
 
 Qi chose to jump off the building due to the suicide of her childhood friend by jumping off a building. She always regretted not being able to save him, so she chose the same way to end her own life.
 
 Qi wrote in her suicide note:
-“My death has nothing to do with anyone, it’s just a choice of mine, please don′t hate others for it. Please live a good, sunny and happy life, and please live a good life for yourself.”
+“My death has nothing to do with anyone, it’s just a choice of mine, please don't hate others for it. Please live a good, sunny and happy life, and please live a good life for yourself.”
```
Comment (Aquaticat) [inline people/qiqi233345/page.en.md:20]: Don't "correct" curly quotes into straight quotes.

File: people/qianyuanakg/page.en.md [hunk 18]
```diff
@@ -22,7 +22,7 @@ Interestingly, she has a lovely sister who bites others (playfully and harmlessl
 I met Qianyuan in November 2022. In my impression, she was a lively high-school student. 
 
 She had dishevelled hair of medium length, which grew over her ears and nearly covered her eyes.
-She always wore a light gray scarf and a light green coat coat.
+She always wore a light grey scarf and a light green coat.
```
Comment (Aquaticat) [inline people/qianyuanakg/page.en.md:25]: Prefer en_US or en_CA over en_GB.

File: people/mone/page.en.md [hunk 19]
```diff
@@ -9,9 +9,9 @@ info:
 
 Mone is a cisgender ally among us. She's quite a cute little girl.
 
-She loves idol groups and singing tv programmes in Japan and South Korea, and often reposts their tweets.
+She loves idol groups and singing TV programmes in Japan and South Korea, and often reposts their tweets.
 
-She also plays Minecraft. In that game, she built a small villa with a cozy room belonging to herself.
+She also plays Minecraft. In that game, she built a small villa with a cosy room belonging to herself.
```
Comment (Aquaticat) [inline people/mone/page.en.md:14]: Prefer en_US or en_CA over en_GB.

File: people/lintong/page.en.md [hunk 20]
```diff
@@ -9,9 +9,9 @@ info:
 
 > Lin Tong is a girl who is a bit insecure.
 > She has a mother who she rarely contacts.
-> Dishes like “scrambled eggs with tomatoes” were her favorites.
+> Dishes like “scrambled eggs with tomatoes” were her favourites.
 > Monster Energy was her drink of choice.
-> Personalitywise,
+> Personality wise,
```
Comment (Aquaticat) [inline people/lintong/page.en.md:14]: Personalitywise is indeed a real word.
https://en.wiktionary.org/wiki/personalitywise

File: people/lintong/page.en.md [hunk 21]
```diff
@@ -9,9 +9,9 @@ info:
 
 > Lin Tong is a girl who is a bit insecure.
 > She has a mother who she rarely contacts.
-> Dishes like “scrambled eggs with tomatoes” were her favorites.
+> Dishes like “scrambled eggs with tomatoes” were her favourites.
```
Comment (Aquaticat) [inline people/lintong/page.en.md:12]: https://en.wiktionary.org/wiki/personalitywise

File: people/donotexist_A/page.en.md [hunk 22]
```diff
@@ -42,7 +42,7 @@ Exist has too many good qualities to list.
 
 Exist helped many trans people in Guangdong to build up confidence by using her social media.
 She shared her experience of how she was able to get a diagnosis of gender dysphoria and get the prescriptions with everyone.
-We believe, if we are able to see Exist one more time, she would share all she saw and learnt during her time in the other world with us.
+We believe, if we are able to see Exist one more time, she would share all she saw and learned during her time in the other world with us.
```
Comment (Aquaticat) [inline people/donotexist_A/page.en.md:45]: Prefer en_US or en_CA over en_GB.

File: people/akasa_musha/page.en.md [hunk 23]
```diff
@@ -8,11 +8,11 @@ Yuxue is a person who craves companionship, which was often mentioned in her twe
 
 She had the side of being a little girl. She used to decorate mangoes with candles. She also did some research on electronics.
 
-Her parents expressed serious disapproval of her transgender behavior at first, but eventually gained a little understanding of her after her efforts. Like other Chinese schools, her school prevented her from carrying HRT medicines to school and keeping her hair long, which may have exacerbated her depression.
+Her parents expressed serious disapproval of her transgender behaviour at first, but eventually gained a little understanding of her after her efforts. Like other Chinese schools, her school prevented her from carrying HRT medicines to school and keeping her hair long, which may have exacerbated her depression.
```
Comment (Aquaticat) [inline people/akasa_musha/page.en.md:11]: Prefer en_US or en_CA over en_GB.

File: people/akasa_musha/page.en.md [hunk 24]
```diff
@@ -8,11 +8,11 @@ Yuxue is a person who craves companionship, which was often mentioned in her twe
...
 
 It is a pity that her friends’ efforts did not stop her depression from worsening.
 In the end, she chose suicide.
 
-After her death, Han Lianyi (a Chinese trans women known for her volunteer suicide prevention work) reflected on her failure to transfer Yuxue to someone else in time when Han was not capable enough for her case; failing to discover Yuxue's obvious signs of suicide in time; and her inadequacy in guiding the rescue of Yuxue at hospital. Han hopes that this failure could help guide future suicide prevention efforts.
+After her death, Han Lianyi (a Chinese transwoman known for her volunteer suicide prevention work) reflected on her failure to transfer Yuxue to someone else in time when Han was not capable enough for her case; failing to discover Yuxue's obvious signs of suicide in time; and her inadequacy in guiding the rescue of Yuxue at hospital. Han hopes that this failure could help guide future suicide prevention efforts.
```
Comment (Aquaticat) [inline people/akasa_musha/page.en.md:16]: transwoman is not a real word. It's also often used as a degrading term to say trans women, somehow, are fundamentally not "women".

trans woman, however, is adj + noun, means a woman who's trans.

Should be "trans woman".

File: people/a2581911655/page.en.md [hunk 25]
```diff
@@ -34,7 +34,7 @@ During the time, she came to Guangdong, and got into [Yantian](https://one-among
 
 After that, she arrived in Shanghai, Zhejiang, and Henan, and met with many friends.
 
-On January 18th, 2024, she ended her journey reluctantly and said that she won't able to see everyone agin until one and a half year later. 
+On January 18th, 2024, she ended her journey reluctantly and said that she won't able to see everyone again until one and a half year later. 
 
-But she couldn't fulfill her appointment. 
+But she couldn't fulfil her appointment.
```
Comment (Aquaticat) [inline people/a2581911655/page.en.md:39]: Prefer en_US or en_CA over en_GB.

File: people/Xu_Yushu/page.en.md [hunk 26]
```diff
@@ -128,10 +128,10 @@ After listening to the advice of the doctor, he told Xu Yushu "Try to forget it,
 Xu Yushu's father actually always tried to understand Xu Yushu.
 He learnt psychology in university but was never taught anything about gender.
 He didn't think his child really had gender dysphoria.
-He said Xu Yushu "never had any weird behaviors" at home,
+He said Xu Yushu "never had any weird behaviours" at home,
```
Comment (Aquaticat) [inline people/Xu_Yushu/page.en.md:131]: Prefer en_US or en_CA over en_GB.

**Review by Aquaticat — CHANGES_REQUESTED** [review]

File: people/Dethelly/page.en.md (hunk 4 above)
Comment (cvyl) [reply people/Dethelly/page.en.md:59]: I had British English in mind. My apologies, it should stay at vacation.

File: people/MizuharaNagisa/page.en.md (hunk 10 above)
Comment (cvyl) [reply people/MizuharaNagisa/page.en.md:19]: Thank you, I thought of that too but wasn't too sure >.<

File: people/akasa_musha/page.en.md (hunk 24 above)
Comment (cvyl) [reply people/akasa_musha/page.en.md:16]: Apologies for my oversight, I was not aware.

**Comment by cvyl** [comment]
Apologies for my oversights. I will make sure to keep in mind the use of American / Canadian English. I hope you can forgive me for my blunders. 🙇‍♀️

**Comment by Aquaticat** [comment]
Great!

**Comment by Twinsherry** [comment]
> Apologies for my oversights. I will make sure to keep in mind the use of American / Canadian English. I hope you can forgive me for my blunders. 🙇‍♀️

Thank you for your work again.

Besides, if you find something may cause misunderstandings, you can contact us and help us improve it!

File: people/yingying/page.en.md (hunk 15 above)
Comment (Twinsherry) [reply people/yingying/page.en.md:13]: > Should this really be past tense? I felt like the meaning is about what she wants to do in the future.

我想了一下，修改者的想法也许是觉得在愿来生做“她（以前）想做（但没做）的事情”，但祈使句的时态确实应该保持将来时。不知道这处修改是否和修改者的母语系统有关。

**Pull request merged**